README.md

```diff
@@ -8,6 +8,28 @@
 ![](https://img.shields.io/github/repo-size/TheAlgorithms/Python.svg?label=Repo%20size&style=flat-square)&nbsp;
 <!--[![Tested on Python 3.7](https://img.shields.io/badge/Tested%20-Python%203.7-blue.svg?logo=python&style=flat-square)]( https://www.python.org/downloads) &nbsp;-->
 
+
+### Description
+
+This repository contains all the algorithms implemented in Python 
+Just copy search the repository for the algorithm you want to implement and use
```

Comment:
`copy` is not required in its current location.

`Just search the repository for the algorithm(s) you want to use and copy them.  You can also improve these algorithms or add new ones by creating pull requests.`

Reply:
yes